### ISSUE TITLE
java.lang.IllegalStateException in DBAdapter

### ISSUE THREAD

**Issue by alexandrudsc** [alexandrudsc/University-timetable-Android#45]
Huawei HUAWEI P smart 2019 (HWPOT-H), Android 

```java
java.lang.IllegalStateException: 
 
  at android.database.sqlite.SQLiteClosable.acquireReference (SQLiteClosable.java:55)
 
  at android.database.sqlite.SQLiteDatabase.queryWithFactory (SQLiteDatabase.java:1330)
 
  at android.database.sqlite.SQLiteDatabase.query (SQLiteDatabase.java:1206)
 
  at android.database.sqlite.SQLiteDatabase.query (SQLiteDatabase.java:1374)
 
  at com.developer.alexandru.orarusv.data.DBAdapter.a (DBAdapter.java:108)
 
  at com.developer.alexandru.orarusv.app_widget.ListRemoteViewsFactory.onDataSetChanged (ListRemoteViewsFactory.java:61)
 
  at android.widget.RemoteViewsService$RemoteViewsFactoryAdapter.onDataSetChanged (RemoteViewsService.java:142)
 
  at com.android.internal.widget.IRemoteViewsFactory$Stub.onTransact (IRemoteViewsFactory.java:50)
 
  at android.os.Binder.execTransact (Binder.java:739)
```

**Comment by alexandrudsc** [comment]
related to #32

**Comment by alexandrudsc** [comment]
added on the 10th
```java
java.lang.IllegalStateException: 
 
  at android.database.sqlite.SQLiteConnectionPool.throwIfClosedLocked (SQLiteConnectionPool.java:1223)
 
  at android.database.sqlite.SQLiteConnectionPool.waitForConnection (SQLiteConnectionPool.java:848)
 
  at android.database.sqlite.SQLiteConnectionPool.acquireConnection (SQLiteConnectionPool.java:413)
 
  at android.database.sqlite.SQLiteSession.acquireConnection (SQLiteSession.java:909)
 
  at android.database.sqlite.SQLiteSession.executeForCursorWindow (SQLiteSession.java:849)
 
  at android.database.sqlite.SQLiteQuery.fillWindow (SQLiteQuery.java:62)
 
  at android.database.sqlite.SQLiteCursor.fillWindow (SQLiteCursor.java:149)
 
  at android.database.sqlite.SQLiteCursor.getCount (SQLiteCursor.java:137)
 
  at android.database.AbstractCursor.moveToPosition (AbstractCursor.java:220)
 
  at android.database.AbstractCursor.moveToNext (AbstractCursor.java:269)
 
  at com.developer.alexandru.orarusv.data.DBAdapter.a (DBAdapter.java:121)
 
  at com.developer.alexandru.orarusv.app_widget.ListRemoteViewsFactory.onDataSetChanged (ListRemoteViewsFactory.java:61)
 
  at android.widget.RemoteViewsService$RemoteViewsFactoryAdapter.onDataSetChanged (RemoteViewsService.java:142)
 
  at com.android.internal.widget.IRemoteViewsFactory$Stub.onTransact (IRemoteViewsFactory.java:50)
 
  at android.os.Binder.execTransact (Binder.java:739)
```

**Comment by alexandrudsc** [comment]
this should be fixed by this a7e979983c2d11f2d5392a978cdf8f5469a4c455   
will reopen if persists
